lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -117,7 +127,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $output->writeln(sprintf('Generated new migration class to "<info>%s</info>" from schema differences.', $path));
     }
 
-    private function buildCodeFromSql(Configuration $configuration, InputInterface $input, array $sql)
+    private function buildCodeFromSql(Configuration $configuration, array $sql, $formatted=false, $lineLength=0)
```

Comment:
no. It needs to be an integer. But any value smaller than 27 will be broken (see the logic line 147)